src/Routing/RouteCollection.php

```diff
@@ -77,6 +77,12 @@ public function add(Route $route, array $options = [])
 
         // Explicit names
         if (isset($options['_name'])) {
+            if (isset($this->_named[$options['_name']])) {
+                throw new MissingRouteException([
```

Comment:
This case: `DuplicateNamedRouteException`

If in general the Router should become more strict now allowing to declare the same Route twice `DuplicateRouteException`